src/plugins/midi-piano/piano-component.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import { Piano } from 'react-piano';
+import React, { useEffect, useState, useRef } from 'react';
+
+export default function PianoComponent(props) {
+
+  const defaultKeyWidth = 29.2;
+  const renderCount = useRef(0);
+  const renderPiano = useRef(false);
```

Comment:
This should be a state variable and could be called more like a flag (now it's name sounds like it is a method), maybe `const [canRenderPiano, setCanRenderPiano] = useState(false);`